input/docs/handbook/commands/index.md

```diff
@@ -1,23 +1,175 @@
 
-`ReactiveCommand` has full support for asynchronous logic and will block re-execution while executing. `ReactiveCommand` guarantees the result of events are delivered to the provided `IScheduler` (which defaults to the main thread scheduler). The executing logic thread safety is the user's responsibility but any result from the logic is guaranteed to arrive on the specified `IScheduler`.
+# Creating commands
 
 A `ReactiveCommand` is created using static factory methods which allows you to create command logic that executes either synchronously or asynchronously. The following are the different static factory methods:
 
 * `CreateFromTask()` - Executes a C# [Task Parallel Library (TPL)](https://docs.microsoft.com/en-us/dotnet/standard/parallel-programming/task-based-asynchronous-programming) Task. This allows use also of the C# [async/await](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/async) operators.
 * `CreateFromObservable()` - Execute the logic using an `IObservable`.
 * `Create()` - Execute a synchronous Func or Action.
+* `CreateCombined()` - Read more on combining commands [here](./combining-commands).
+
+All of these methods will parameterize the resulting `ReactiveCommand` to be the return result of the method (i.e. if your async method returns `Task<string>`, your Command will be `ReactiveCommand<TInput, string>`). This means, that subscribing to the command itself returns the results of the async method as an Observable.
+
+# Synchronous commands
+
+If your command is not CPU-intensive or I/O-bound then it probably makes sense to provide synchronous execution logic. You can do so by creating a command via `ReactiveCommand.Create`:
+
+```cs
+// Creates a command with synchronous execution logic
+// which is always available for execution.
+ReactiveCommand<Unit, Unit> command = ReactiveCommand.Create(
+    () => Console.WriteLine("A reactive command is invoked!")
+);
+```
+
+# Asynchronous commands
+
+One of the most important features of `ReactiveCommand` is its built-in facilities for orchestrating asynchronous operations, commands will block re-execution while executing. `ReactiveCommand`s are fully integrated into the Reactive Extensions framework, providing an `.IsExecuting` property (of type `IObservable<bool>`) which tells you whether the command is currently executing. This is often useful if you want to trigger activity animations or you want to prevent other commands from executing while the command is executing.
+
+It is important to know, that ReactiveCommand itself as an `IObservable` will never complete or OnError - errors that happen in the async method will instead show up on the `ThrownExceptions` property. If it is possible that your async method can throw an exception, you should subscribe to `ThrownExceptions` or the exception will be rethrown on the UI thread.
```

Comment:
Maybe mention you can override the Default Exception Handler thing, maybe link to the Page that talks about that.